### PR TITLE
Add jquery-cookiebar@1.0.5 w/ npm auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #10441 

@cdnjs/intern3 Please help me review this PR. Thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/missing-code/jquery-cookiebar
 * Official website (optional, not the repository): http://www.primebox.co.uk/projects/cookie-bar/
 * NPM package url (optional): https://www.npmjs.com/package/jquery-cookiebar
 * License and its reference: MIT [ref](https://github.com/missing-code/jquery-cookiebar/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 1
   - Count of stars: 1
   - Count of forks: 3
 * NPM download stats (optional):
   - Downloads in the last day: 19
   - Downloads in the last week: 111
   - Downloads in the last month: 608

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.cookiebar/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "keywords": [
+    "cookie",
+    "cookiebar",
+    "jquery",
+    "cookielaw",
+    "eucookies"
+  ],
+  "author": "Stefan Kecskes <info@skey.uk> (https://skey.uk)",
+  "license": "MIT",
+  "homepage": "git://github.com/missing-code/jquery-cookiebar#readme",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.cookiebar/package.json]: Wrong homepage ...

File: ajax/libs/jquery.cookiebar/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.cookiebar",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.cookiebar/package.json]: Why not `jquery-cookiebar` as its repo and npm package name?

**Comment by PeterDaveHello** [comment]
@hare1039 in this case, you should request changes, not comment.

File: ajax/libs/jquery-cookiebar/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "cookie",
+    "cookiebar",
+    "jquery",
+    "cookielaw",
+    "eucookies"
+  ],
+  "author": "Stefan Kecskes <info@skey.uk> (https://skey.uk)",
+  "license": "MIT",
+  "homepage": "http://www.primebox.co.uk/projects/cookie-bar/",
+  "npmName": "jquery-cookiebar",
```
Comment (hare1039) [inline ajax/libs/jquery-cookiebar/package.json]: The title is git auto-update, but inside `package.json` is npm auto-update. Also the commit massage too.

Comment (PeterDaveHello) [inline ajax/libs/jquery-cookiebar/package.json]: @clairetsai818 you should reply after you fix it.

Comment (clairetsai818) [inline ajax/libs/jquery-cookiebar/package.json]: @hare1039 thanks! I revised package.json. @PeterDaveHello Please help me review this!

File: ajax/libs/jquery-cookiebar/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "keywords": [
+    "cookie",
+    "cookiebar",
+    "jquery",
+    "cookielaw",
+    "eucookies"
+  ],
+  "author": "Stefan Kecskes <info@skey.uk> (https://skey.uk)",
+  "license": "MIT",
+  "homepage": "http://www.primebox.co.uk/projects/cookie-bar/",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-cookiebar/package.json]: Should be https://www.primebox.co.uk/projects/jquery-cookiebar/, @cdnjs/intern3 don't forget to make sure everything is good.

**Comment by clairetsai818** [comment]
@PeterDaveHello I revised `homepage`, please help me review this. Thanks!

**Comment by PeterDaveHello** [comment]
Both title and commit message are wrong ...

**Comment by clairetsai818** [comment]
@PeterDaveHello I revised commit message, please help me review this. Thanks!

**Comment by PeterDaveHello** [comment]
@clairetsai818 the file permission is wrong, I'll fix for you this time, you should be careful with it next time.

**Pull request merged**